core/state-db/src/pruning.rs

```diff
@@ -67,24 +67,24 @@ fn to_journal_key(block: u64) -> Vec<u8> {
 impl<BlockHash: Hash, Key: Hash> RefWindow<BlockHash, Key> {
 	pub fn new<D: MetaDb>(db: &D) -> Result<RefWindow<BlockHash, Key>, Error<D::Error>> {
 		let last_pruned = db.get_meta(&to_meta_key(LAST_PRUNED, &()))
-			.map_err(|e| Error::Db(e))?;
-		let pending_number: u64 = match last_pruned {
+			.map_err(Error::Db)?;
+		let pending_number_: u64 = match last_pruned {
 			Some(buffer) => u64::decode(&mut buffer.as_slice())? + 1,
 			None => 0,
 		};
-		let mut block = pending_number;
+		let mut block = pending_number_;
 		let mut pruning = RefWindow {
 			death_rows: Default::default(),
 			death_index: Default::default(),
-			pending_number: pending_number,
+			pending_number: pending_number_,
```

Comment:
Should the fixes here not be this instead?

```suggestion
			pending_number,
```

(same for all the similar ones)